README.md

```diff
@@ -145,3 +145,7 @@ A curated list of awesome niche job boards.
 ### Free & Open Source
 
 * [Free & Open Source Jobs](https://www.fossjobs.net/)
+
+## Tech Jobs with verified relocation packages
```

Comment:
Could we put this board under the `Tech` heading? You can then change the entry to be something like:

```
* [Relocate.me](https://relocate.me/) - Verified relocation packages
```

TYVM!